tests/server/routes/category-routes-tests.js

```diff
@@ -306,12 +306,13 @@ describe('Categories Route', function () {
       });
     });
 
-  it('edit a category and get a 200 message', function (done){
+  it('edit a category and get a 304 message', function (done){
```

Comment:
This was switched to a 200 status. Just make sure the test text matches the test itself